### PR TITLE
(#862) Add documentation about Temporary Groups

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description Of Changes

This adds some information about what a Temporary Group is,
what the intention for such a group is, and where this Group will
be visible.

## Motivation and Context

To Document Temporary Groups.

## Testing

* [ ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

~* [ ] Minor documentation fix (typos etc.).~
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
~* [ ] New documentation page added.~

## Change Checklist

~* [ ] Requires a change to menu structure (top or left hand side)/~
~* [ ] Menu structure has been updated~

## Related Issue

Reference #862

### CODE REVIEW

**Comment by AdmiringWorm** [comment]
@st3phhays you may want to update the Automatic Deployment Plan documentation to reference this section of the Group Page, at least on those you have mentioned Temporary Groups.

File: input/en-us/central-management/usage/website/groups.md [hunk 1]
```diff
@@ -31,6 +31,17 @@ The  `All Computers (Automatic Group)` will have a blue lock icon next to the th
...
+## Temporary Groups
+
+> :choco-info: **NOTE**
+>
+> This feature is available in Chocolatey Centtral Management starting with version 0.12.0.
+
+A Temporary Group is a Group that has been created for one or more specific circumstance, for instance as part of creating an Automatic Deployment Plan.
+These Groups are normally not viewable from anywhere, other than the Deployment Plans where they are being used, and will be cleaned up on an interval when they are no longer needed.
+
+A Temporary Group can also not be Edited, Deleted or Duplicated manually by a user, as well as no other Group can have such a Group as a Sub/Child Group.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/groups.md:43]:
```suggestion
A Temporary Group can also not be Edited, Deleted or Duplicated manually by a user, as well as no other Group can have such a Group as a Sub-Group.
```

We refer to this as a Sub-Group elsewhere, so let's stick with that.

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/groups.md [hunk 2]
```diff
@@ -31,6 +31,17 @@ The  `All Computers (Automatic Group)` will have a blue lock icon next to the th
...
+## Temporary Groups
+
+> :choco-info: **NOTE**
+>
+> This feature is available in Chocolatey Central Management starting with version 0.12.0.  
+
+A Temporary Group is a Group that has been created for one or more specific circumstance, for instance as part of creating an Automatic Deployment Plan.
+These Groups are normally not viewable from anywhere, other than the Deployment Plans where they are being used, and will be cleaned up on an interval when they are no longer needed.
+
+A Temporary Group can also not be Edited, Deleted or Duplicated manually by a user, as well as no other Group can have such a Group as a Sub-Group.
```
Comment (corbob) [inline input/en-us/central-management/usage/website/groups.md:43]:
```suggestion
A Temporary Group can also not be Edited, Deleted, or Duplicated manually by a user. As well, no other Group can have such a Group as a Sub-Group.
```
Just a minor grammar fixup I think.

**Review by corbob — CHANGES_REQUESTED** [review]

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**